packages/model/src/reminder.ts

```diff
@@ -0,0 +1,48 @@
+import { ListResponse } from './common';
+import { ResearchOutputResponse } from './research-output';
+
+type ReminderArea = 'Research Output' | 'Events';
+
+export const reminderTypes = [
+  'ResearchOutputPublished',
+  'ResearchOutputShare',
+] as const;
+
+export type ReminderType = typeof reminderTypes[number];
+
+interface Reminder {
+  area: ReminderArea;
+  type: ReminderType;
+  data: unknown;
+}
+
+export interface ResearchOutputPublishedReminder extends Reminder {
+  area: 'Research Output';
+  type: 'ResearchOutputPublished';
+  data: {
```

Comment:
Do we need to wrap this in "data"?  Maybe it could be called source?

Reply:
im not sure, i dont think having all the extra props flat out is a great idea, feels cleaner this way

Reply:
I am hoping we can just include the source entity flat against the object and maybe a JQL query when required to identify exactly what was changed if required